src/run_validation.py

```diff
@@ -10,19 +10,31 @@
 
 from system import console
 from validation_workflow.validation_args import ValidationArgs
+from validation_workflow.validation_test_runner import ValidationTestRunner
 
 
 def main() -> int:
-
     args = ValidationArgs()
 
     console.configure(level=args.logging_level)
     logging.getLogger("urllib3").setLevel(logging.WARNING)
 
-    dist = args.DISTRIBUTION_MAP.get(args.distribution, None)
-    dist.download_artifacts(projects=args.projects, version=args.version)
-
-    return 0
+    dist = None
+    if args.platform == 'docker':
```

Comment:
Thanks for the comment, @gaiksaya. The reason we placed the docker as a platform was because what it shown on opensearch.org/downloads.html where it defines Linux, Docker, Windows, and FreeBSD as Platform. We can certainly update it/docker to be one of the distribution.